tasties_app/tests/test_category.py

```diff
@@ -0,0 +1,124 @@
+    return (category_1, category_2, category_3)
+
+
+@pytest.fixture()
+def categorized_recipes(recipes, categories):
+    """
+    This fixture assigns Categories to Recipes
+
+    Args:
+        recipes (fixture): fixture 'recipes'
+        categories (fixture): fixture 'categories'
+
+    Returns:
+        tuple<Recipe>: The Recipes generated by 'recipes', assigned with
+                       Categories generated by 'categories'
+    """
+    recipes[0].categories.add(categories[0])
+    recipes[0].categories.add(categories[1])
+    recipes[1].categories.add(categories[1])
+    return (recipes[0], recipes[1])
+
+
+class TestCategoryModel:
+    @pytest.mark.django_db
+    def test_create_categories(self, categories, recipes):
+        """
+        Verifies fixtures 'categories' and 'recipes' properly
+           saved their data as expected
+
+        Args:
+            categories (fixture): fixture 'categories'
+            recipes (fixture): fixture 'recipes'
+        """
+        all_categories = Category.objects.all()
+        all_recipes = Recipe.objects.all()
+        for category in categories:
+            assert category in all_categories
+        for recipe in recipes:
+            assert recipe in all_recipes
+
+    @pytest.mark.django_db
+    def test_get_recipes_by_category(self, categories, recipes, categorized_recipes):
+        """
+        Verifies method 'get_recipes_by_category' provides intended functionality
+
+        Args:
+            categories (fixture): fixture 'categories'
+            recipes (fixture): fixture 'recipes'
+            categorized_recipes (fixture): fixture 'categorized_recipes'
+        """
+        # define test variables
+        category_1 = categories[0]
+        category_2 = categories[1]
+        category_3 = categories[2]
+        recipe_1 = recipes[0]
+        recipe_2 = recipes[1]
+        # generate recipe_by_category QuerySets
+        recipes_by_category_1 = Category.get_recipes_by_category(category_1)
+        recipes_by_category_2 = Category.get_recipes_by_category(category_2)
+        recipes_by_category_3 = Category.get_recipes_by_category(category_3)
+        # assert expected results
+        assert len(recipes_by_category_3) == 0  # category with no associated recipes will return empty set
+        assert recipe_2 not in recipes_by_category_1
+        assert recipe_1, recipe_2 in recipes_by_category_2
+        assert len(Category.get_recipes_by_category("Cheese")) == 0  # invalid parameters will result in empty set
+
+    @pytest.mark.django_db
+    def test_category_name_uniqueness(self, categories):
+        """
+        Verifies that a new Category cannot be saved, if it has the same name as an existing Category
+
+        Args:
+            categories (fixture): test data generated by fixture 'categories'
+        """
+        category_4 = Category(category_name="Breakfast")
+        from django.db import IntegrityError
```

Comment:
According to [pep8](https://peps.python.org/pep-0008/#imports), imports are always at the top of the file.